config/environments/development.rb

```diff
@@ -131,6 +129,29 @@
   routes.default_url_options ||= {}
   routes.default_url_options[:script_name]= ''
 
+  # https://tasdikrahman.me/2020/07/07/structured-logging-in-rails/
+  config.lograge.formatter = Lograge::Formatters::Json.new
+  config.lograge.enabled = true
+  config.lograge.base_controller_class = ['ActionController::Base']
+  config.lograge.custom_options = lambda do |event|
+    {
+      request_time: Time.now,
+      application: Rails.application.class,
+      process_id: Process.pid,
+      host: event.payload[:host],
+      remote_ip: event.payload[:remote_ip],
```

Comment:
Much of this is already in `config/application.rb` maybe we should centralize it all there?

Reply:
👍